docs/docs-components/SandpackExample.js

```diff
@@ -243,11 +243,14 @@ export default function SandpackExample({
           import { Box, ColorSchemeProvider } from 'gestalt';
           import App from "./App";
 
+          const html = document.querySelector('html');
```

Comment:
nice fix

Reply:
I tried to override `index.html` but looks like Sandpack doesn't allow it. So I had to improvise 😬